projects/core/src/i18n/translation-chunk.service.ts

```diff
@@ -47,7 +48,7 @@ export class TranslationChunkService {
         }.${key}'.`
       );
     });
-    console.warn(
+    logger.warn(
```

Comment:
please revert wrapping logs which were not checked by if(isDevMode) like that 
```suggestion
    console.warn(
```